packages/typespec-ts/test/integration/azurecorescalar.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { assert } from "chai";
+import AzureCoreScalarClientFactory, {
+  AzureCoreScalarClient
+} from "./generated/azure/core-scalar/src/index.js";
+describe("Azure Core Saclar Rest Client", () => {
+  let client: AzureCoreScalarClient;
+
+  beforeEach(() => {
+    client = AzureCoreScalarClientFactory({
+      allowInsecureConnection: true
+    });
+  });
+
+  it("should get an Azure Location value", async () => {
+    try {
+      const result = await client
+        .path("/azure/core/scalar/azureLocation")
+        .get();
+      assert.strictEqual(result.status, "200");
+      assert.strictEqual(result.body, "eastus");
+    } catch (err) {
+      assert.fail(err as string);
+    }
+  });
+
+  it("should put an Azure Location value", async () => {
+    try {
+      const result = await client.path("/azure/core/scalar/azureLocation").put({
+        body: "eastus",
+        headers: {
+          "content-type": "text/plain"
```

Comment:
I think this is expected, as cadl-ranch is expecting string here https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/azure/core/scalar/mockapi.ts#L15